common/util/file_util_test.cc

```diff
@@ -61,6 +64,18 @@ TEST(FileUtil, Stem) {
   EXPECT_EQ(file::Stem("/foo/bar"), "/foo/bar");
 }
 
+TEST(FileUtil, JoinPath) {
+  EXPECT_EQ(file::JoinPath("foo", ""), "foo/");
+  EXPECT_EQ(file::JoinPath("", "bar"), "/bar");
```

Comment:
Could we also add
`EXPECT_EQ(file::JoinPath("/", "bar"), "/bar");`

Reply:
Done, also with a few more variants.